jaxlib/lapack.pyx

```diff
@@ -358,8 +358,10 @@ def jax_getrf(c, a):
             batch_dims + (m, n),
             (num_bd, num_bd + 1) + tuple(range(num_bd - 1, -1, -1))),
       ))
+  return tuple(c.GetTupleElement(out, i) for i in range(3))
```

Comment:
Could use `xla_destructure(c, out)` from `jax.interpreters.xla` here, not that it saves much typing.

Reply:
Probably better to avoid jaxlib -> jax dependencies, though...